target/linux/ipq40xx/image/generic.mk

```diff
@@ -28,6 +28,17 @@ define Device/UbiFit
 	IMAGE/nand-sysupgrade.bin := sysupgrade-tar | append-metadata
 endef
 
+define Build/qsdk-ipq-app-gpt
+	cp $@ $@.tmp 2>/dev/null || true
```

Comment:
Why "|| true"? Wouldn't we want to fail this entirely if the copy can't be made?

Reply:
TBH, I don't know, but I can test the variant that does not have `|| true`, and if it works fine, I will submit that instead.

Reply:
Can you suggest the normal way to do? I can try it out to see if it works. As mentioned, this is originally from Li, so I can't answer what this recipe looks like this :D